### ISSUE TITLE
WebAssembly.Table.get(): invalid index 584944 into funcref table of size 1000

### ISSUE THREAD

**Issue by zoriya** [ThaUnknown/jassub#32]
I updated all my npm dependencies and encountered this issue, I can't tell why it happens
```
Uncaught (in promise) RangeError: WebAssembly.Table.get(): invalid index 584944 into funcref table of size 1000
    at ie (jassub-worker.js:2:9156)
    at Wr (jassub-worker.js:15:9810)
    at jassub-worker.wasm:0xa18b4
    at jassub-worker.wasm:0xa68eb
    at jassub-worker.wasm:0xea152
    at Dr (jassub-worker.js:15:10018)
    at jassub-worker.js:15:10373
```

Here is the context, since the stack trace is probably not that helpful:
```js
	function xt(t, r, n) {
        f.hasOwnProperty(t) || ge("Replacing nonexistant public symbol"),
        f[t].overloadTable !== void 0 && n !== void 0 ? f[t].overloadTable[n] = r : (f[t] = r,
        f[t].argCount = n)
    }
    function Lt(t, r, n) {
        var i = dynCalls[t];
        return n && n.length ? i.apply(null, [r].concat(n)) : i.call(null, r)
    }
    var be = [];
    function ie(t) {
        var r = be[t];
        return r || (t >= be.length && (be.length = t + 1),
        be[t] = r = ue.get(t)), // Here
        r
    }
    function Ht(t, r, n) {
        if (t.includes("j"))
            return Lt(t, r, n);
        var i = ie(r).apply(null, n);
        return i
    }
    function Bt(t, r) {
        var n = [];
        return function() {
            return n.length = 0,
            Object.assign(n, arguments),
            Ht(t, r, n)
        }
    }
```

After searching a bit, it seems that the error is really similar to this one: https://github.com/jellyfin/jellyfin-web/issues/4794 but on a modern version of chrome/firefox:

Chrome version: Version 117.0.5938.149
Firefox version: Version 118.0.1

If you need more context for this, you can take a look at this branch where I updated packages: https://github.com/zoriya/Kyoo/compare/master...fix/things

**Comment by ThaUnknown** [comment]
I have no clue, that's compiled code that's throwing the error, reproduction steps would be better

**Comment by zoriya** [comment]
Turns out this happens for any next 14 application, I don't know if I bundle jassub wrong or what they changed between version 13 and 14 to make this bug appears but here is a minimal reproduction repo: https://codesandbox.io/p/sandbox/elastic-antonelli-qrjfj8

That's simply a next-js template with a jassub component in index.tsx and it is bundled via a copy-weback-plugin (since import.meta.url is not available on next nor are import "file?url")

**Comment by ThaUnknown** [comment]
I'm fairly sure next is gonna be to blame here?
Try reproducing this statically without NextJS

**Comment by zoriya** [comment]
I am totally sure that Next is to blame but it's not like I can remove it so I would like to make this work. Do you have an idea how to proceed to debug this? I don't think I did anything wrong with how jassub is served or initialized.
Do you have some debugging steps I could take to pinpoint the issue?

**Comment by ThaUnknown** [comment]
man you'll be happy to hear this: try 1.7.11

**Comment by ThaUnknown** [comment]
fixed in 1.7.11, thanks for the report, I have no clue how I overlooked this!!!
![image](https://github.com/ThaUnknown/jassub/assets/6506529/218d38f7-918a-42e7-a19a-c9fa833b04e2)

**Comment by zoriya** [comment]
I think we were both waiting for the CI x) Thanks for the quick fix!
